chainerx/_docs/routines.py

```diff
@@ -447,6 +447,30 @@ def _docs_indexing():
 .. seealso:: :func:`numpy.take`
 """)
 
+    _docs.set_doc(
+        chainerx.where,
+        """where(condition, x, y)
+Return elements chosen from x or y depending on condition.
+
+Args:
+    condition (~chainerx.ndarray): Where True, yield x, otherwise yield y.
+    x (~chainerx.ndarray): Values from which to choose.
+    y (~chainerx.ndarray): Values from which to choose.
+
+Returns:
+    :func:`~chainerx.ndarray`: An array with elements
+    from x where condition is True, and elements from y elsewhere.
+
+Note:
+    x, y and condition need to be broadcastable to some shape.
```

Comment:
```suggestion
    ``x``, ``y`` and ``condition`` need to be broadcastable to some shape.
```

Reply:
By the way, I think this note is not needed because other docs also don't have such a note.

Reply:
Oh. Will remove it.